content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -108,25 +111,25 @@ We worked on an initial proposal that included the following:
 
 This proposal was presented to the OpenTelemetry Demo SIG during one of the
```

Comment:
The SIG has been referred to as "demo SIG" throughout, suggest applying the lowercase 'd' here for consistency. Not a blocking item!